app/va/identifier.py

```diff
@@ -1,5 +1,7 @@
 from enum import Enum
 
+from app.constants import STATUS_REASON_NO_PROFILE
+
 
 # TODO: Refactor this file to have actual classes that contain suffix, oid properties etc instead of all these dicts
```

Comment:
we had a ticket for this and then closed it. can you please remove this comment while you're here?